src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -165,6 +169,13 @@ public TopicRepository getDefaultTopicRepository() throws TopicRepositoryExcepti
         return topicRepositoryHolder.getTopicRepository(defaultStorage);
     }
 
+    public EventConsumer createEventConsumer(final String clientId, final List<NakadiCursor> positions)
+            throws NakadiException, InvalidCursorException {
+        final MultiTimelineEventConsumer result = new MultiTimelineEventConsumer(clientId, this, timelineSync);
+        result.reassign(positions);
```

Comment:
I do not want to have it inside constructor, because there will be 2 ways of initialization. And reinitialization I will need in any case for subscriptions